nwamugo/politico-react
# 166334689-feature(Landing-page): user can see landing page

What does this PR do?
It adds the Landing Page of the application

Description of Task to be completed?
Users can view landing page of the application

How should this be manually tested?
- Clone the repo
- Run npm install
- Run npm run watch

What are the relevant pivotal tracker stories?(if applicable)
#166334689 Users can see a landing page

Screenshots
![image](https://user-images.githubusercontent.com/41264503/58648440-9b95c300-8301-11e9-8731-87fe02734c65.png)
